plugins/modules/monitoring/newrelic_deployment.py

```diff
@@ -13,24 +13,26 @@
 ---
 module: newrelic_deployment
 author: "Matt Coddington (@mcodd)"
-short_description: Notify newrelic about app deployments
+short_description: Notify New Relic about app deployments
 description:
-   - Notify newrelic about app deployments (see https://docs.newrelic.com/docs/apm/new-relic-apm/maintenance/deployment-notifications#api)
+  - Notify New Relic about app deployments (see https://docs.newrelic.com/docs/apm/new-relic-apm/maintenance/record-monitor-deployments/)
 options:
   token:
     type: str
     description:
-      - API token, to place in the x-api-key header.
+      - API token, to place in the Api-Key header.
```

Comment:
I think that comma is redundant
```suggestion
      - API token to place in the Api-Key header.
```